server/database/queries/user/addOrderRequest.js

```diff
@@ -0,0 +1,13 @@
+const connection = require('../../config/connection');
+
+const addOrderRequest = ({ id, providerId, description }) => {
```

Comment:
if you use userId instead of id, I think it will be more readable.
```suggestion
const addOrderRequest = ({ userId, providerId, description }) => {
```